pipelines/pipeline_katello.yml

```diff
@@ -28,7 +28,7 @@
     - foreman_repositories
     - katello_repositories
     - role: foreman_client_repositories
-      when: katello_version in ['3.9', 'nightly']
+      when: katello_version is version('3.9', '>=')
```

Comment:
Is nightly considered newer than 3.9?

Reply:
yes (you acked PRs with that already :P https://github.com/theforeman/forklift/pull/877)

Reply:
funnily, this only works on python2, so I will check for nightly too :/